packages/dev/core/src/Meshes/GaussianSplatting/gaussianSplattingMesh.ts

```diff
@@ -13,13 +13,21 @@ import { Constants } from "core/Engines/constants";
 import { Tools } from "core/Misc/tools";
 import "core/Meshes/thinInstanceMesh";
 import type { ThinEngine } from "core/Engines/thinEngine";
+import { Frustum } from "core/Maths/math.frustum";
 
 interface DelayedTextureUpdate {
     covA: Float32Array;
     covB: Float32Array;
     colors: Float32Array;
     centers: Float32Array;
 }
+
+export interface BoundingVolume {

```

Comment:
I think `BoundingVolume` is a bit too generic a name, maybe prefix it with `GaussianSplatting` or something similar?